README.md

```diff
@@ -168,25 +190,28 @@ zen: {
 * env vars that are service specific, these are read next
 * local setup specific variables, these are read last
 
-Settings at any level will override existing settings, so for example anything you set in your local setup will override any previous setting.
+Settings at any level will override existing settings, so for example anything
+you set in your local setup will override any previous setting.
 
 See [system.js](system.js) for global, system and service environment variables.
 
 To set local environment variables, set them in the `docker-compose.yml`:
 
 ### Testing
 
-To create a clean test database add the env variable `ZENTEST=true` to the testdata container before
-running `docker-compose up localdev`. This 
+To create a clean test database add the env variable `ZENTEST=true` to the testdata
+container before running `docker-compose up localdev`. This will load a clean data
+for e2e tests
```

Comment:
e2e are severely broken. Plan is to move to cp-zen-frontend's e2e, but we need to make changes around testdata to actually run them end-to-end. Right now they're basically integration tests. Might be best to remove this I guess?